test/adql/db/TestFunctionDef.java

```diff
@@ -273,7 +273,7 @@ public void testCompareToFunctionDef(){
 		// TEST :: With a function having the same name, but a different return type: [EQUAL}
 		assertEquals(0, def1.compareTo(new FunctionDef("fct1", new DBType(DBDatatype.INTEGER))));
 
-		// TEST :: With a function having the same name, but 2 parameters: [LESS (4 characters: ø against 1010)]
+		// TEST :: With a function having the same name, but 2 parameters: [LESS (4 characters: 0 against 1010)]
```

Comment:
Actually, no: by `ø` I really meant no character instead of having the characters `1010` in addition inside the comparison form of the second function. This test is actually comparing the comparison form of the two functions:

* `fct1() -> VARCHAR` is serialized for comparison only into `fct1`
* `fct1(foo INTEGER, foo INTEGER) -> INTEGER` is serialized for comparison only into `fct1100100`

So comparing the two functions here, means comparing the strings `fct1` and `fct1100100` which gives `-6` (i.e. 6 characters are different).

_Note that the comment is saying `4 characters` but it should now be `6 characters` and that instead of `1010` we should have `100100`. But the `ø` character is still valid and means "empty string".